data/config-schema/src/main/kotlin/net/corda/schema/configuration/ConfigDefaults.kt

```diff
@@ -13,9 +13,6 @@ package net.corda.schema.configuration
 object ConfigDefaults {
     const val JDBC_DRIVER = "org.postgresql.Driver"
     const val DB_POOL_MAX_SIZE = 10
-    const val RECONCILIATION_PERMISSION_SUMMARY_INTERVAL_MS_DEFAULT: Long = 60000L
-    const val RECONCILIATION_CPK_WRITE_INTERVAL_MS_DEFAULT: Long = 10000L
-    const val RECONCILIATION_CPI_INFO_INTERVAL_MS_DEFAULT: Long = 10000L
```

Comment:
I think these defaults need to be in the json schema.

Reply:
Good catch, yes indeed, will create the reconciliation schema and add the defaults. For now making sure things work when porting boot config to the database to be reconciled.

Reply:
Done